### DISCUSSION TITLE
Installing on Dell PowerEdge with Perc S110 RAID controller

### DISCUSSION THREAD

**Discussion by jcurtiss7492** [Security-Onion-Solutions/securityonion#12420] [2.4] [answered]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16 GB

### Storage for /

200 GB

### Storage for /nsm

800 GB

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I'm working on a PoC for my company and I'm excited to try out Security Onion.  I am fairly new to SO and have some experience with Linux.

I'm trying to install SO on a repurposed Dell PowerEdge T320 that uses a Dell PERC S110 Raid controller.

I have (4) 500 GB drives configured in a RAID 10 array.

When I get to the first part of the install using the SO ISO, it asks for the locations to install the OS and /NSM storage, but it only lists and I can only select the (4) 500GB drives individually, not as the RAID 10 array.

Multiple drives detected.  Let's answer some questions.
Device     Size
sda          465GB
sdb          465GB
sdc          465GB
sdd          465GB
There are (4) available disk(s)
Which device would you like to use as the operating system filesystem (must be larger than 99GB):

How can I get SO to recognize the RAID10 array to install on?

Thanks!

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by petiepooo** [comment, marked as answer]
According to the PERC S110 [User Guide](https://dl.dell.com/manuals/all-products/esuprt_ser_stor_net/esuprtl_adapters/poweredge-rc-s110_user's%20guide_en-us.pdf), it is a software RAID controller (aka FakeRAID) only supported under Windows.  Your best bet is probably to make all disks non-RAID on the PERC, and create a MD mirror of the first two disks on install then afterwards create a mirror of the second pair, make it a PV, add it to the existing VG, and grow your nsm LV and filesystem to use it.  You will need to be comfortable with the OS installer's disk configuration helper and LVM2 tools to do that.

Alternatively, if this is only a POC, you could throw redundancy out of the window and give yourself nearly 2T for /nsm.. at least until a disk fails.  That's a simpler matter of installing to one disk, then adding the three remaining as PVs to grow the /nsm LV and fs.

**Comment by jcurtiss7492** [comment]
Thank you so much for the quick answer!  I should've realized it was SW RAID.
